halo/evmstaking2/keeper/keeper_internal_test.go

```diff
@@ -90,48 +91,53 @@ func TestInsertAndDeleteEVMEvents(t *testing.T) {
 	}
 }
 
-func TestDeliverDelegate(t *testing.T) {
+func TestDelivery(t *testing.T) {
```

Comment:
would be great if we can test edge cases as well, i.e. delivery errors and panics.